src/Countries/Tunisia.php

```diff
@@ -0,0 +1,2266 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use RuntimeException;
+
+class Tunisia extends Country
+{
+    /**
+     * @var array<int, array<int, array{name: string,date: string,slug: string}>> $hijriHolidays
+     */
+    protected array $hijriHolidays = [
+        "1970" => [
```

Comment:
isn't a solution like Turkey possible here?

https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11

Reply:
It's not a good solution I know I found the turkey code approved by the repo team so I adapted the code with the other Hijri holidays

Reply:
We can use an external hijri converter package

Reply:
> isn't a solution like Turkey possible here?
> 
> https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11

Tomorrow I will apply the same Turkey code with more holidays